src/main/java/com/github/distributedtracing/ContinuationLocal.java

```diff
@@ -0,0 +1,170 @@
+package com.github.distributedtracing;
+
+public final class ContinuationLocal<T> {
+    private static InheritableThreadLocal<Object[]> localCtx =
```

Comment:
@mariusae was it a conscious decision to use ThreadLocal insted of InheritableThreadLocal, in the first place?